osmosis-1/osmosis-1.assetlist.json

```diff
@@ -597,6 +597,36 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/xki.svg"
       },
       "coingecko_id": "ki"
+    },
+    {
+      "description": "The native token of Secret Network",
+      "denom_units": [
+        {
+          "denom": "ibc/0954E1C28EB7AF5B72D24F3BC2B47BBB2FDF91BDDFD57B74B99E133AED40972A",
```

Comment:
Confirm that this is the correct denom:
```
$ osmosisd q ibc-transfer denom-trace 0954E1C28EB7AF5B72D24F3BC2B47BBB2FDF91BDDFD57B74B99E133AED40972A
denom_trace:
  base_denom: uscrt
  path: transfer/channel-88
```
Channel query:
```
$ osmosisd q ibc channel end transfer channel-88
channel:
  connection_hops:
  - connection-1244
  counterparty:
    channel_id: channel-1
    port_id: transfer
  ordering: ORDER_UNORDERED
  state: STATE_OPEN
```
Connection Query:
```
$ osmosisd q ibc connection end connection-1244
connection:
  client_id: 07-tendermint-1588
  counterparty:
    client_id: 07-tendermint-2
    connection_id: connection-1
    prefix:
      key_prefix: aWJj
  delay_period: "0"
  state: STATE_OPEN
```
Client Query:
```
$ osmosisd q ibc client state 07-tendermint-1588
client_state:
  '@type': /ibc.lightclients.tendermint.v1.ClientState
  allow_update_after_expiry: true
  allow_update_after_misbehaviour: true
  chain_id: secret-4
  frozen_height:
    revision_height: "0"
    revision_number: "0"
  latest_height:
    revision_height: "989065"
    revision_number: "4"
  max_clock_drift: 35s
  proof_specs:
```

This appears to be correct